shell/browser/osr/osr_web_contents_view.cc

```diff
@@ -107,18 +107,18 @@ content::DropData* OffScreenWebContentsView::GetDropData() const {
 }
 
 gfx::Rect OffScreenWebContentsView::GetViewBounds() const {
-  return GetView() ? GetView()->GetViewBounds() : gfx::Rect();
+  if (auto* view = GetView())
+    return view->GetViewBounds();
+  return {};
```

Comment:
I might be missing something, but should we return a `gfx::Rect()` here instead of `{}`?

Reply:
This is a C++ idiom saying "return a default-constructed instance of whatever the return type of this function is." -- in this case, it's the same as saying `return gfx::Rect{}`.

The compiler deduces the return type from the declaration at the beginning of the function, and the empty brace-closed initializer list means to use the defaults when building it.

This idiom is good because:

- It's terse
- It cleanly expresses an _idea_: "return the default value." This is its the main value.
- It's futureproof: if have a function with a bunch of checks that call `return {};` when the check fails, and then we change that function's return type, then we won't have to change all those bits.

Also used upstream in base, content, ui, etc. :+1:  We should use it more often